### ISSUE TITLE
Value directives placed after !important are ignored

### ISSUE THREAD

**Issue by asmadi** [MohammadYounes/rtlcss#91]
I'm trying to change this font 
font-family: "Droid Sans", sans-serif !important /*!rtl:prepend:"Droid Arabic Kufi",*/;
But for some reason the "!rtl" is not working and even without it if any property has !important the converting will fail

Thank You.

**Comment by MohammadYounes** [comment]
Yes, this is mainly caused by postcss treating the comment as part of the important property rather than the value. I opened an [issue](https://github.com/postcss/postcss/issues/966) over at postcss repo.  

For the time being and as a **workaround**, place the directive any where inside the declaration value :

> Don't place it at the beginning, due to this [bug](https://github.com/postcss/postcss/issues/598)

```css
font-family: "Droid Sans", /*!rtl:prepend:"Droid Arabic Kufi",*/sans-serif!important;
```


Hopefully, this will get fixed by postcss [soon](https://github.com/postcss/postcss/issues/966#issuecomment-278433955)!

**Comment by asmadi** [comment]
Awesome, the work around working just fine

Thank you!

**Comment by MohammadYounes** [comment]
Fixed in ee9dc53237e73b725f800a9a7a392cfc53f67e4a
